doc/release/release_0.20.rst

```diff
@@ -0,0 +1,658 @@
+- Add missing copyrights to LICENSE.txt and use formatting according to SPDX identifiers
+  (`#6419 <https://github.com/scikit-image/scikit-image/pull/6419>`_).
+- Include ``skimage.morphology.footprint_from_sequence`` in the public API documentation
+  (`#6555 <https://github.com/scikit-image/scikit-image/pull/6555>`_).
+- Correct note about return type in the docstring of ``skimage.exposure.rescale_intensity``
+  (`#6582 <https://github.com/scikit-image/scikit-image/pull/6582>`_).
+- Stop using the ``git://`` connection protocol and remove references to it
+  (`#6201 <https://github.com/scikit-image/scikit-image/pull/6201>`_, `#6283 <https://github.com/scikit-image/scikit-image/pull/6283>`_).
+- Update scikit-image's mailing addresses to the new domain discuss.scientific-python.org
+  (`#6255 <https://github.com/scikit-image/scikit-image/pull/6255>`_).
+- Remove references to deprecated mailing list in ``doc/source/user_guide/getting_help.rst``
+  (`#6575 <https://github.com/scikit-image/scikit-image/pull/6575>`_).
+- Use "center" in favor of "centre", and "color" in favor of "colour" gallery examples
+  (`#6421 <https://github.com/scikit-image/scikit-image/pull/6421>`_, `#6422 <https://github.com/scikit-image/scikit-image/pull/6422>`_).
+- Replace reference to ``api_changes.rst`` with ``release_dev.rst``
+  (`#6495 <https://github.com/scikit-image/scikit-image/pull/6495>`_).
+- Clarify header pointing to notes for latest version released
+  (`#6508 <https://github.com/scikit-image/scikit-image/pull/6508>`_).
+- Add missing spaces to error message in ``skimage.measure.regionprops``
+  (`#6545 <https://github.com/scikit-image/scikit-image/pull/6545>`_).
+- Apply codespell to fix common spelling mistakes
+  (`#6537 <https://github.com/scikit-image/scikit-image/pull/6537>`_).
+- Add missing space in math directive in normalized_mutual_information's docstring
+  (`#6549 <https://github.com/scikit-image/scikit-image/pull/6549>`_).
+- Fix lengths of docstring heading underline in ``skimage.morphology.isotropic_`` functions
+  (`#6628 <https://github.com/scikit-image/scikit-image/pull/6628>`_).
+- Fix plot order due to duplicate examples with the file name ``plot_thresholding.py``
+  (`#6644 <https://github.com/scikit-image/scikit-image/pull/6644>`_).
+- Get rid of numpy deprecation warning in gallery example ``plot_equalize``
+  (`#6650 <https://github.com/scikit-image/scikit-image/pull/6650>`_).
+- Fix swapping of opening and closing in gallery example ``plot_rank_filters``
+  (`#6652 <https://github.com/scikit-image/scikit-image/pull/6652>`_).
+- Get rid of numpy deprecation warning in gallery example ``in plot_log_gamma.py``
+  (`#6655 <https://github.com/scikit-image/scikit-image/pull/6655>`_).
+
+Other and development related updates
+-------------------------------------
+.. Everything invisible to a user of scikit-image or not covered in the above section. These changes probably not interesting to users.
```

Comment:
```suggestion
```

Reply:
The remaining comments all relate to autogenerated text. I will address this in a future PR after 0.20.0rc0.